frontend/src/HomePage/HomePage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import axios from 'axios';
+import NavBar from '../Navbar/NavBar';
+
+class HomePage extends React.Component {
+	state = { details: [] };
```

Comment:
```suggestion
	const [posts, setPosts] = useState([]);
```
Let's utilize the react hooks